webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue

```diff
@@ -161,20 +161,64 @@ export default {
       this.drawer = true;
     },
     saveGroup(event) {
+      const startWithRegex =  /^\d/;
       const regex =  /^[a-zA-Z0-9-_]+$/;
```

Comment:
Could you combine those two regex in one ?

Reply:
First one is for: Name starts with number: "The "Group Name" field must start with a character and must not contain special characters."
Second one is for: special characters.
And each one is used apart.

Reply:
Done